packages/bundlers/default/src/DefaultBundler.js

```diff
@@ -435,7 +438,11 @@ function deduplicate(bundleGraph: MutableBundleGraph) {
           bundle.hasAsset(asset) &&
           bundleGraph.isAssetReachableFromBundle(asset, bundle)
         ) {
-          bundleGraph.removeAssetGraphFromBundle(asset, bundle);
+          if (isSiblingDedupe) {
```

Comment:
I don't think we should be treating these differently

Reply:
these tests fail if we do not treat them differently 😞: https://dev.azure.com/devongovett/devongovett/_build/results?buildId=15550&view=logs&j=81a0a40d-e1c6-55f1-066f-a450d17357d3&t=9ab6c9af-8c9a-51e6-60c5-00f9d8182a4a&l=1404

Reply:
That makes sense — they're no longer reachable because the context changes across those sibling bundles.